src/main/resources/reference.conf

```diff
@@ -607,7 +613,7 @@ resourceTypes = {
         }
       }
       notebook-user = {
-        roleActions = ["launch_notebook_cluster", "create_persistent_disk", "create_kubernetes_app", "add_child"]
+        roleActions = ["launch_notebook_cluster", "create_persistent_disk", "create_kubernetes_app", "add_child", "list_children"]
```

Comment:
This should be added to the `owner` role above instead

Reply:
My thinking was that `add_child` > `list_children` (and `notebook-user` is included in the `owner` role), so it made sense for it to be here.